build.sh

```diff
@@ -52,17 +52,22 @@ function build_fluentbit() {
   cd submodules/fluent-bit
   mkdir -p build
   cd build
-  cmake .. -DCMAKE_INSTALL_PREFIX=$subagentdir/fluent-bit -DFLB_HTTP_SERVER=ON -DFLB_DEBUG=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWITHOUT_HEADERS=ON
+  cmake .. -DCMAKE_INSTALL_PREFIX=/usr/lib/google-cloud-ops-agent \
```

Comment:
Took me a while to figure this out. `CMAKE_INSTALL_PREFIX` and `CMAKE_INSTALL_BINDIR` will cause the binary to be put at `/usr/lib/google-cloud-ops-agent/./fluent-bit`. `FLB_SHARED_LIB=OFF` causes it to not generate `libfluent-bit.so` at all. Can you add this in a comment for the next person?